### DISCUSSION TITLE
Page Actions Don't Appear for Non-Admin Users

### DISCUSSION THREAD

**Discussion by myoongbpl** [requarks/wiki#6423] [Error / Bug Report] [answered]
![pageactions](https://github.com/requarks/wiki/assets/114519381/68aaaf06-c128-4087-9e1d-86876c097806)
Page Actions buttons does not appear for non-admin groups. My permissions have read, write, and manage pages enabled. I managed to find that enabling write:groups will make the Page Actions button appear again.

**Comment by NGPixel** [comment, marked as answer]
You need to give page rules with write permissions as well. Giving global permissions alone is not enough. See https://docs.requarks.io/groups
